app/src/cose.h

```diff
@@ -76,6 +96,91 @@ namespace scitt::cose
     std::optional<int64_t> notary_authentic_signing_time;
     std::optional<int64_t> notary_expiry;
 
+    bool is_present(const std::variant<int64_t, std::string>& label) const
```

Comment:
Surely there is a better way to do this but I couldn't find a simple way to access a struct's fields by name